cli/app.py

```diff
@@ -0,0 +1,157 @@
+# Copyright (c) Nordic Semiconductor ASA
```

Comment:
I think these should be places in bin/ folder. Perhaps app itself should be located to source folder and here is just simple entrypoint for it so coverage is calculated

Reply:
Lets do in follow-up pr.